build/configs/imxrt1050-evk/loadable_elf_apps/defconfig

```diff
@@ -572,7 +587,7 @@ CONFIG_LPUART1_2STOP=0
 # CONFIG_DISABLE_MOUNTPOINT is not set
 # CONFIG_DISABLE_PSEUDOFS_OPERATIONS is not set
 CONFIG_FS_READABLE=y
-# CONFIG_FS_WRITABLE is not set
+CONFIG_FS_WRITABLE=y
```

Comment:
This should be disabled for bin manger. It causes ftl error.

Reply:
That issue is fixed because of 2MB kernel ram